tests/e2e/driver/CheReporter.ts

```diff
@@ -88,6 +90,8 @@ class CheReporter extends mocha.reporters.Spec {
       }
       // await preferencesHandler.setConfirmExit(AskForConfirmationType.never);
       // await preferencesHandler.setTerminalType(TerminalRendererType.dom);
+
+      monacoPageObjects.initPageObjects('1.71.0', '1.37.0', vscodeExtensionTesterLocators.getLocatorsPath(), driver.get(), 'google-chrome');
```

Comment:
Hard coded version values are hard to locate and impossible to change when run tests against different versions of VS Code Editor.

What do think about extracting the values as a constant to setup them using environment variables?

Reply:
good idea

Reply:
done.